spec/features/active_admin_feature_spec.rb

```diff
@@ -31,7 +31,7 @@
   context 'when spo' do
     before do
       signin_spo_user
-      ci = create(:case_information, team: nil)
+      ci = create(:case_information, probation_service: 'Scotland', team: nil)
```

Comment:
It might be nice to turn this into a trait 'nilteam' so that it doesn't look weird that we are picking (validly) probation_service Scotland for no obvious reason